chrome/content/zotero/xpcom/server.js

```diff
@@ -466,6 +466,7 @@ Zotero.Server.DataListener.prototype._processEndpoint = Zotero.Promise.coroutine
 			} else if(this.contentType === "multipart/form-data") {
 				let boundary = /boundary=([^\s]*)/i.exec(this.header);
 				if (!boundary) {
+					Zotero.debug('Invalid boundary: ' + this.header);
```

Comment:
Doesn't really matter, but should use a higher debug level here — probably 1 if this is equivalent to an error. Or if this is actually worth logging to the console, could just use `Zotero.logError()`.